### PR TITLE
add table convertor to componentstatus

### PR DESCRIPTION
**Pull request by zhouya0**
**What type of PR is this?**
> /kind bug

**What this PR does / why we need it**:
Add table convertor to componentstatus.  After delete the use of internalversion in kubectl, it need to add table convertor to componentstatus (others already have). 

**Which issue(s) this PR fixes**:

Fixes # https://github.com/kubernetes/kubectl/issues/762

**Does this PR introduce a user-facing change?**:
```release-note
Add table convertor to componentstatus.
```

### CODE REVIEW

**Review by soltysh — APPROVED** [review]
/kind bug
/sig cli
/sig apimachinery
/priority important-longterm
/ok-to-test
/lgtm

File: pkg/registry/core/componentstatus/rest.go [hunk 1]
```diff
@@ -45,6 +49,12 @@ func NewStorage(serverRetriever func() map[string]*Server) *REST {
 	}
 }
 
+// ConvertToTable converts the result to the table.
+func (rs *REST) ConvertToTable(ctx context.Context, object runtime.Object, tableOptions runtime.Object) (*metav1beta1.Table, error) {
+	tableConvertor := printerstorage.TableConvertor{TableGenerator: printers.NewTableGenerator().With(printersinternal.AddHandlers)}
```
Comment (liggitt) [inline pkg/registry/core/componentstatus/rest.go]: this constructs/registers all handlers on every API request?

Comment (liggitt) [reply pkg/registry/core/componentstatus/rest.go]: fix in https://github.com/kubernetes/kubernetes/pull/85309

**Pull request merged**